app/views/lists/_list.html.erb

```diff
@@ -0,0 +1,7 @@
+<%= div_for(@lists) do |list| %>
```

Comment:
Totally agree, and render collection will be a bit faster as well because the partial doesn't need to parse multiple times.